source/_docs/custom-certificates.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Custom Certificates on the Global CDN
+description: For contract customers who need to retain custom TLS certificates.
+contributors: [ari, alexfornuto]
+earlyaccess: true
+earlynote: A white glove Custom Certificate Concierge Service in now available and limited to  Legacy SSL Customers with a Pantheon contract, including Enterprise, EDU+, Pantheon One, Elite, and Reseller customers.
+---
+
+## Eligibility
+
+Access to this service is limited to existing Legacy SSL customers with a Pantheon contract.
+
+## Why upgrade to the Global CDN?
+
+If your site is using our Legacy SSL service it's on deprecated, legacy infrastructure, served from a single server in the central US. When you upgrade to the Global CDN you'll see faster performance, with content  delivered from over 50 points of presence (POPs) around the world.
+
+If your site uses Legacy SSL, it's also on an outdated TLS configuration. The Global CDN is configured to only use TLS 1.2 and no weak 3DES cipher. On the legacy infrastructure, your site isn't as fast, secure, or as resilient as it could be.
+
+## Upgrade to Global CDN with Free & Automated HTTPS
+
+If you just haven’t gotten around to upgrading, consider using our managed HTTPS, which includes automated certificate management, leveraging Let’s Encrypt certificates. As a convenience, when you upgrade to managed HTTPS you’ll never have to worry about an expired certificate again. As long as your domain is pointed to Pantheon, we will automatically renew the certificates required to keep your site secure. [Upgrade](/docs/https/) to get best-in-class encryption and an A+ grade from SSL Labs.
+
+## Upgrade to Global CDN with Manual Custom Certificate
```

Comment:
Shorter header with common vocab, make an h3 like:
> ### Custom HTTPS

Reply:
+1 for shorter...Custom Certificates